admin_guide/router.adoc

```diff
@@ -73,6 +73,41 @@ To view a router log, run the `oc logs` command on the pod. Since the router is
 running as a plug-in process that manages the underlying implementation, the log
 is for the plug-in, not the actual HAProxy log.
 
+To view the logs generated by HAProxy you need to start a syslog server and pass
+the location to a router pod using two environment variables.
+
+.Router Syslog Variables [[syslog-vars]]
+[cols="3a,8a",options="header"]
+|===
+
+|Environment Variable | Description
+
+|*`ROUTER_SYSLOG_ADDRESS`*
+|The IP address of the SYSLOG server. Port *514* is the
+default if no port is specified.
+
+|*`ROUTER_LOG_LEVEL`*
```

Comment:
Sorry, typo'd this earlier. :( Should be asterisk outside of backtick:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference